src/amo/utils/addonVersions.ts

```diff
@@ -2,32 +2,43 @@ import fetch from "node-fetch";
 import * as vscode from "vscode";
 
 import { showErrorMessage } from "./processErrors";
+import { makeAuthHeader } from "./requestAuth";
 import constants from "../../config/config";
-import { addonVersion } from "../types";
+import { addonVersion, errorMessages } from "../types";
 
 export async function getAddonVersions(input: string, next?: string) {
   const slug: string = input.includes("/")
     ? input.split("addon/")[1].split("/")[0]
     : input;
   const url = next
     ? next
-    : `${constants.apiBaseURL}addons/addon/${slug}/versions/`;
-  const response = await fetch(url);
+    : `${constants.apiBaseURL}addons/addon/${slug}/versions?filter=all_with_unlisted`;
```

Comment:
`all_with_deleted` includes unlisted versions too.  

But we might have a problem here - the permission `ADDONS_VIEW_DELETED` isn't necessarily held by all reviewers (nor the permission for reviewing unlisted either!) and the permission check in the API will error if that filter is requested without the appropriate permission.  I'd file a follow-up for this because we'll want Andreas' opinion on what to do - if there will be scenarios when reviewers are using this plugin without `ADDONS_VIEW_DELETED` (or `ADDONS_REVIEW_UNLISTED`) then it may need to be a setting in the plugin.